core/modules/widgets/navigator.js

```diff
@@ -288,7 +290,8 @@ NavigatorWidget.prototype.makeDraftTiddler = function(targetTitle) {
 				"draft.title": targetTitle,
 				"draft.of": targetTitle
 			},
-			this.wiki.getModificationFields()
+			this.wiki.getModificationFields(),
+			tiddler == null ? defaultFields : {}
```

Comment:
We use `===` like: `tiddler === null ? defaultFields : {}`

Reply:
Good call out, I've updated the condition to include both null and undefined with a strict equality comparator. The code did not work as expected with just `tiddler === null`.